### ISSUE TITLE
[Bug]: "dispaly info" in attribute table show nothing

### ISSUE THREAD

**Issue by gioman** [3liz/lizmap-web-client#3674]
### What is the bug?

In the attribute table of a layer (that has the "popup" option active), the "display info" panel is empty

![image](https://user-images.githubusercontent.com/1951107/236669713-9aba480c-7abb-4477-9d43-a323a5f10f95.png)

### Steps to reproduce the issue

Publish a layer in LMWC with the popup option active, and also with the attribute table available.

Open the table, select a record/feature and click on "display info" icon.

### Versions

QGIS LTR on Server and Desktop, Latest version of LMWC, LM plugin and LM server plugin

### Check Lizmap plugin

- [X] I have done the step just before in the Lizmap QGIS desktop plugin before opening this ticket. Otherwise, my ticket is not considered valid and might get closed.

### QGIS server version, only if the section above doesn't mention the QGIS Server version

3.28

### Operating system

Ubuntu 22.04, desktop and server

### Browsers

Firefox, Chrome

### Browsers version

Latest

**Comment by Antoviscomi** [comment]
@gioman 
in my LWC 3.6.2 on ubuntu 22.04 all seems works fine
![Schermata del 2023-05-08 13-56-02](https://user-images.githubusercontent.com/12639444/236817782-3da40a27-c7a2-4013-9d38-3f53f18e19fe.png)

**Comment by gioman** [comment]
> in my LWC 3.6.3 on ubuntu 22.04 all seems works fine

@Antoviscomi I have the same setup, and here is not working. Maybe is something project specific. I'm happy to let someone have a look if necessary.

**Comment by gioman** [comment]
Found the reason: a field that is excluded from the WMS/WFS capabilities then should not be used as "primary key" in LM plugin when it comes to te configs for a layer attribute table.

I think that fields that are excluded from the WMS/WFS capabilities should not be shown at all in Lizmap plugin... feel free to move this to the LM plugin tracker, as suggestion for improvement.

**Comment by gioman** [comment]
@Gustry fair enough. I think anyway that a small mouseover warning in the plugin could help the users: "do not use for primery key a field that is excluded from WMS/WFS capabilities".

**Comment by Gustry** [comment]
I added it in the ticket #389 already
